README.md

```diff
@@ -115,7 +115,14 @@ make install prefix=/usr/local
 Prerequisites for compilation are:
 
 * `make`
-* [Go 1.9+](http://golang.org/doc/install)
+* [Go 1.11+](http://golang.org/doc/install)
```

Comment:
This project builds on Go 1.9 just fine; we use it in CI.

Reply:
ah, interesting -- i made this change since i was unable to build without module support and saw that that was introduced after 1.9 but i see it was backported 👍